### PR TITLE
Add blog post: Getting Started with Solidus Starter Frontend

### PR DESCRIPTION
**Pull request by gsmendoza**

### CODE REVIEW

File: source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+title: Getting Started with Solidus Starter Frontend
+date: 2021-11-25
+tags: frontend
+author: George Mendoza
+cover_image: /blog/2021/11/25/getting-started-with-solidus-starter-frontend/home.png
```
Comment (gsmendoza) [inline source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown:6]: @kennyadsl I may need some help to make this cover image nicer:

image

File: source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/80-checkout-confirm.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Confirm page</figcaption>
+    </figure>
+    <figure class="figure">
+      <img src="/blog/2021/11/25/getting-started-with-solidus-starter-frontend/90-checkout-complete.png" class="figure-img img-fluid" />
+      <figcaption class="figure-caption">Checkout - Complete page</figcaption>
+    </figure>
+  </div>
+</div>
```
Comment (gsmendoza) [inline source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown:110]: @kennyadsl We can customize this carousel to better present the screenshots. Let me know which format in https://kenwheeler.github.io/slick/#demos is ideal.

File: source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown (hunk 1 above)
Comment (kennyadsl) [reply source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown:6]: @mfrecchiami ping 🙂

File: source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+In addition to generating views for authentication and storefront functionality,
+the application template will also generate
+
+* Mailers with their corresponding views
+* [View components](https://viewcomponent.org) for complex presentational logic
+* Unit and integrated specs for testing the frontend
```
Comment (kennyadsl) [inline source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown:117]: `integrated` -> `integration`?

**Review by kennyadsl — COMMENTED** [review]
Thanks, this is nice! 

I have the feeling we could improve this with some notes about the next steps for the starter frontend in our plans. Can we add a final paragraph about this please?

File: source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown (hunk 1 above)
Comment (mfrecchiami) [reply source/blog/2021-11-25-getting-started-with-solidus-starter-frontend.html.markdown]: @gsmendoza added to my ToDo list, I'll tackle it tomorrow!

**Comment by gsmendoza** [comment]
@kennyadsl Appreciate if you can review again. Changes:

1. Updated the date stamp
2. Added link to screencast
3. Added next steps for the starter frontend in our plans
4. Minor changes to the content to make them read better

**Review by kennyadsl — APPROVED** [review]
Thanks @gsmendoza! 


@mfrecchiami I think this may be better with a proper cover image, instead of the plain screenshot. Do you have time to create one for us? If you don't in the next days, no problem, just let us know and we'll merge this as is and we can add a better cover later. 🙏

**Comment by mfrecchiami** [comment]
@Alberto Vena ***@***.***> I'm OOO until Monday! I can design
a cover the next Tuesday.

Il ven 17 giu 2022, 11:23 Alberto Vena ***@***.***> ha
scritto:

File: source/blog/2022-06-16-getting-started-with-solidus-starter-frontend.html.markdown [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+* Unit and integrated specs for testing the frontend
+
+## What's next for Solidus Starter Frontend?
+
+For Solidus 4, SolidusStarterFrontend will become the recommended option for
+building the storefront of a new Solidus app. We're planning to remove
+SolidusFrontend from the solidus meta-package gem and move its code to a new
+repo. If your app depends on SolidusFrontend, you'll need to explicitly add
+`solidus_frontend` to your Gemfile in order to continue using it.
+
```
Comment (kennyadsl) [inline source/blog/2022-06-16-getting-started-with-solidus-starter-frontend.html.markdown:126]: Last thing while we wait for the cover: we could add that we are still working on a solution to make the extensions compatible with the new frontend.

**Comment by gsmendoza** [comment]
@mfrecchiami Thank you for the cover image!

@kennyadsl I have also added a short note about making the extensions compatible with Starter Frontend. Appreciate if you can take a look!

**Review by kennyadsl — APPROVED** [review]

**Pull request merged**